client/my-sites/plugins/marketplace/index.tsx

```diff
@@ -0,0 +1,44 @@
+export type MarketplaceThemeType = {
+	colors: { textColorLight: string; borderColorLight: string };
+	breakpoints: {
+		desktopUp: string;
+		tabletUp: string;
+		bigPhoneUp: string;
+		smallPhoneUp: string;
+	};
+	weights: {
+		bold: string;
+		normal: string;
+	};
+	fonts: {
+		body: string;
+	};
+	fontSize: {
+		small: string;
+	};
+};
+
+const theme: MarketplaceThemeType = {
```

Comment:
Little surprised to see a new theme declaration here — don't we have one that just works across Calypso?

Reply:
Sadly no but I am using css variables wherever possible. Our theme information is mostly embedded in css variables AFAIK.